sentinel-core/src/main/java/com/alibaba/csp/sentinel/util/function/Tuple2.java

```diff
@@ -31,11 +31,24 @@ public Tuple2(R1 r1, R2 r2) {
         return new Tuple2<R2, R1>(this.r2, this.r1);
     }
 
+    @Override
+    public boolean equals(Object o) {
```

Comment:
The `hashCode` method should be provided as well.

Reply:
Thanks for your help,done